week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,87 @@
-// Write your code here
+let slides = [
+  "https://images.pexels.com/photos/235621/pexels-photo-235621.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500",
+  "https://images.pexels.com/photos/259620/pexels-photo-259620.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500",
+  "https://images.pexels.com/photos/355411/pexels-photo-355411.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500",
+];
+// console.log(slides[0].src);
+
+const body = document.querySelector("body");
+body.style.backgroundColor = "pink";
+
+const divBackButton = document.querySelector(".backButton");
+const divForwardButton = document.querySelector(".forwardButton");
+const divPicture = document.querySelector(".picture");
+const divAutoBack = document.querySelector(".autoBack");
+const divAutoForward = document.querySelector(".autoForward");
+const stopButton = document.querySelector(".stopButton");
+const inputField = document.querySelector(".input");
+const backButton = document.createElement("button");
+const forwardButton = document.createElement("button");
+const autoForwardButton = document.createElement("button");
+const autoBackButton = document.createElement("button");
+
+divAutoBack.appendChild(autoBackButton);
+autoBackButton.innerText = "Auto Back";
+divAutoForward.appendChild(autoForwardButton);
+autoForwardButton.innerText = "Auto Forward";
+divForwardButton.appendChild(forwardButton);
+forwardButton.innerText = "Forward";
+divBackButton.appendChild(backButton);
+backButton.innerText = "Back";
+const img = document.createElement("img");
+divPicture.appendChild(img);
+
+let timeBetweenSlides = 2000;
+inputField.addEventListener("input", (e) => {
```

Comment:
Nice work on using the user input to change the carousel speed - however there is no instruction / no indication to the user what that input is for. I had to look at your code to figure it out 😉 Remember to keep the user in mind when designing an app.